examples/influxDB2-nifi/README.md

```diff
@@ -0,0 +1,145 @@
+
+It also seems that changing the version of [Nifi](https://nifi.apache.org/docs/nifi-docs/html/getting-started.html#downloading-and-installing-nifi) (1.12.1 and 1.15.2 tested) doesn't have impact on the InfluxDB connectors.
+Only the version 1.X of influxDB is taken into account.
+
+The only solution found is adding the new processor [updating the Nifi chart](https://github.com/cetic/helm-nifi#use-custom-processors).
+After Nifi reloading, the new libraries are activated.
+
+Installing Fadi
+--
+To run the example at least Nifi, Traefik and InfluxDB must be installed.
+
+### Preparing the installation
+
+Before installing Fadi, the Fadi chart must be customized with the extra libraries of Nifi. 
+If you don't use Nifi v1.12.1 (Fadi currently uses this version), 
+[download](https://github.com/influxdata/nifi-influxdb-bundle#installation) and update the library that fits your needs 
+in the `nifi-influx-nar` folder.
+
+The file `value.yaml` will install Fadi with Nifi, Traefik and InfluxDB. It will also 
+customize the Nifi chart with the downloaded libraries.
+
+### Running the installer
+
+1. in `values.yaml`file, 
+   1. update the Nifi field `path: "/Users/<YOUR_PATH>/nifi-influx-nar"`
+   2. update the InfluxDB password 
+   ```
+      password: "<YOUR_PASSWORD>"
+      token: "<YOUR_PASSWORD>"
+   ```
+2. run `./deploy.sh`
+
+### Testing
+
+The current Nifi version (v 1.12.1) cannot be reached by a port-forwarding.
+Treafik must be used. To run Treafik, an ip must be provided.
+
+if you are using minikube on a local cluster:
+1. run `minikube tunnel` to associate an ip to your cluster (LB to Treafik)
+2. configure your `/etc/hosts` file
+```
+<IP_TUNNEL> nifi.test.local
+```
+3. connect yourself to http://nifi.test.local/nifi/ to reach nifi
+4. forward the port of influxDB 
+
+Running the use case
+--
+
+At this point your must be able to connect yourself to Nifi and InfluxDB.
+
+### Importing data into InfluxDB
+
+Data could be imported with a file or using a copy-paste function.
+
+On the lateral bar choose `Data` and create a bucket. 
+Once done, you could import data with the source tab (line protocol).
+For the use-case we used `earthquake` data. The file is [here](./sample-data/influxDB-earthquake.csv).
+
+Once data is imported you can run the following query:
+
+```
+from(bucket: "testfs")
+  |> range(start: -30d, stop: -1h)
+  |> filter(fn: (r) => r["_measurement"] == "earthquake")
+  |> filter(fn: (r) => r["_field"] == "alert")
+```
+
+### Getting influx data with Nifi
+
+Take care to always using a nifi plugin compatible with influxDB 2.
+
+I get data from InfluxDB with the `GetInfluxDatabaseRecord_2` processor.
+This processor use 2 controllers:
+* a controller to manage the connection to the database (`standardInfluxDatabaseService_2`). 
+  This controller setup the url connection and the InfluxDB AccessToken.![img.png](images/img.png)
```

Comment:
* please use a meaningful image name, img.png is too vague
*  setup --> configures